Remove executable that don't get built from texmath.cabal, so
their dependencies can be discarded.
Index: haskell-texmath-0.1.1/texmath.cabal
===================================================================
--- haskell-texmath-0.1.1.orig/texmath.cabal	2010-01-07 19:05:12.000000000 +0100
+++ haskell-texmath-0.1.1/texmath.cabal	2010-01-07 19:05:29.000000000 +0100
@@ -50,22 +50,3 @@

     Ghc-Options:         -Wall
     Ghc-Prof-Options:    -auto-all
-
-Executable testTeXMathML
-    Main-is:             testTeXMathML.hs
-    Ghc-Options:         -Wall
-    Ghc-Prof-Options:    -auto-all
-    if flag(test)
-      Buildable:         True
-    else
-      Buildable:         False
-
-Executable texmath-cgi
-    Main-is:             cgi/texmath-cgi.hs
-    Build-depends:       cgi, json
-    Ghc-Options:         -Wall
-    Ghc-Prof-Options:    -auto-all
-    if flag(cgi)
-      Buildable:         True
-    else
-      Buildable:         False
